amazon_cloud_code_generator/data/plugins/module_utils/utils.py

```diff
@@ -176,6 +176,44 @@ def boto3_tag_list_to_ansible_dict(
     )
 
 
+def recursive_diff(dict1: Dict, dict2: Dict) -> Dict:
```

Comment:
Why not just use `ansible.module_utils.common.dict_transformations.recursive_diff`?

Reply:
Right! Replaced, thanks!